boards/xtensa/intel_adsp_cavs15/intel_adsp_cavs15.yaml

```diff
@@ -5,7 +5,37 @@ arch: xtensa
 toolchain:
   - xcc
   - zephyr
+supported:
+  - dma
 testing:
   ignore_tags:
-     - net
-     - bluetooth
+    - cmsis_dsp
+    - cmsis_rtos
+    - net
+    - bluetooth
+    - zdsp
+    - can
+    - uart
+    - filesystem
+    - onoff
+    - tinycrypt
+    - hwinfo
+    - test_framework
+    - shell
+    - sensor
+    - sensors
+    - disk
+    - led
+    - zbus
+    - chre
+    - nanopb
+    - smf
+    - compression
+    - dmic
```

Comment:
should we be ignoring dmic?

Reply:
yes, in sof (and for intel) we only care about the dai peripheral interface for all things audio, dmic isn't used